src/store.js

```diff
@@ -0,0 +1,26 @@
+import { createStore } from "@reduxjs/toolkit";
+const initialState = {
+  amount: 0,
+  loan: 0,
+  purpose: "",
```

Comment:
what are these states for... are they also related to the book library project

Reply:
No related to the book library,
It's a practice code to text if Redux installation was successful